front-end/src/components/icons/hamburger.jsx

```diff
@@ -0,0 +1,19 @@
+const Hamburger = () => {
+    return(
+        <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="50"
+            height="60"
```

Comment:
```suggestion
            width="40"
            height="50"
```
Just to make things smaller and more proportional, especially to the changes I suggest in AnalysisPage